src/sql_mock/column_mocks.py

```diff
@@ -8,6 +8,7 @@ class ColumnMock:
     Attributes:
         dtype (str): The data type of the column.
         default: The default value for the column.
+        nullable: Indicator whether the column can be null
```

Comment:
A real nitpick:
The docstring doesn't match ordering that the class attributes are defined